src/PrestaShopBundle/Resources/config/services/core/localization.yml

```diff
@@ -24,3 +24,84 @@ services:
         arguments:
             - '@prestashop.adapter.legacy.configuration'
             - '@prestashop.core.foundation.version'
+
+    # Localization
+    prestashop.core.localization.cache.adapter:
```

Comment:
Theses services are already declared in `config/services/common/services_prod.yml`